components/brave_wallet/browser/fil_requests.cc

```diff
@@ -101,7 +114,12 @@ absl::optional<std::string> getSendTransaction(const std::string& signed_tx) {
   dict.Set("params", std::move(params));
   dict.Set("id", 1);
 
-  return GetJSON(dict);
+  auto json = GetJSON(dict);
+  // Use substring replace instead of deserializing signed_tx to prevent
+  // long to double convertion.
+  base::ReplaceFirstSubstringAfterOffset(&json, 0, "\"{signed_tx}\"",
+                                         signed_tx);
+  return json;
```

Comment:
Can we assume there could be no other appearances of `{signed_tx}` in json?

Reply:
Same question. Could there be a key with an arbitrary value? And if so, are we sure that `{signed_tx}` that we want to replace will alway be the first in the JSON body?

Reply:
Fields are listed in the method, so there shouldn't be such field

Reply:
this is part of what I'm getting at here https://github.com/brave/brave-core/pull/18511/files#r1210199905
this seems very hacky and fragile and I'm fairly certain we already have a better way to handle this